engine/src/main/java/org/destinationsol/game/screens/CameraKbControl.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.game.screens;
+
+import org.destinationsol.GameOptions;
+import org.destinationsol.ui.SolUiControl;
+
+import java.util.List;
+
+class CameraKbControl {
+
+    private static SolUiControl controlUp;
+    private static SolUiControl controlDown;
+    private static SolUiControl controlLeft;
+    private static SolUiControl controlRight;
+
+    CameraKbControl(GameOptions gameOptions, List<SolUiControl> controls) {
+        controlUp = new SolUiControl(null, false, gameOptions.getKeyUp()); // Dirty solution, but on mixed controls, it is uncontrollable
+        controls.add(controlUp);
+        controlDown = new SolUiControl(null, false, gameOptions.getKeyDown());
+        controls.add(controlDown);
+        controlLeft = new SolUiControl(null, false, gameOptions.getKeyLeft());
+        controls.add(controlLeft);
+        controlRight = new SolUiControl(null, false, gameOptions.getKeyRight());
+        controls.add(controlRight);
+    }
+
+    boolean isDown() {
```

Comment:
It might be better to rename this series of methods to something like `isDownControlOn` or `isDownControlEnabled` to be more descriptive.

Reply:
Here I am trying to stay consistent with the rest of the codebase.
(applies for the following 7 comments too)

Reply:
Since this method has been newly introduced in this PR, I'd argue that naming should be fixed here itself. However, if you feel comfortable tackling this in another PR, I should have no problems.